fsw/cfe-core/ut-stubs/ut_sb_stubs.c

```diff
@@ -158,6 +158,35 @@ int32 CFE_SB_CreatePipe(CFE_SB_PipeId_t *PipeIdPtr, uint16 Depth,
     return status;
 }
 
+/*****************************************************************************/
+/**
+** \brief CFE_SB_DeletePipe stub function
+**
+** \par Description
+**        This function is used to mimic the response of the cFE SB function
+**        CFE_SB_DeletePipe.
+**
+** \par Assumptions, External Events, and Notes:
+**        None
+**
+** \returns
+**        Returns CFE_SUCCESS.
```

Comment:
Unless configured not to?

Reply:
I agree with that.
Is that what this means?
```
** \returns
**        Returns either a user-defined status flag or CFE_SUCCESS.
```
Which is the only example I could see that would be comparable.

Reply:
Yes, that's a better description.